packages/schematics/angular/workspace/files/package.json.template

```diff
@@ -8,6 +8,16 @@
     "watch": "ng build --watch --configuration development"<% if (!minimal) { %>,
     "test": "ng test"<% } %>
   },
+  "prettier": {
```

Comment:
In general, I am not a fan of having tools configured in the package manifest. I’d rather have the additional config file which is clearer.

Thoughts @clydin & @dgp1130 ?

Reply:
My thinking here was that creating a dedicated file may be more confusing for people who don't actively use `prettier`. It feels a bit less boilerplate-y. But I don't have super strong opinions either way.

Reply:
That’s a valid point.

Reply:
Sorry, I wasn't talking about changing the behavior. I was talking about silently ignoring a config file that's clearly "meant" to be used (it's called prettierrc after all) and not warning the user about it. Printing a warning should hopefully not break anything and would remove this confusion?